nutrition_ai/lib/screens/authenticated/settings.dart

```diff
@@ -112,7 +116,7 @@ class _SettingsScreenState extends State<SettingsScreen> {
               const SizedBox(height: 20),
               SelectInput(
                 controller: exerciseFrequencyController,
-                defaultValue: user.exerciseFrequency.toString(),
+                defaultValue: (user.exerciseFrequency).isNotEmpty ? user.exerciseFrequency : 'none',
```

Comment:
Same here. This could be replaced with `user.exerciseFrequency ?? 'none'`.

Reply:
same as first comment